libbeat/logp/logger.go

```diff
@@ -0,0 +1,31 @@
+package logp
```

Comment:
The old internal `Logger` type is renamed to `logger`. We now export a `*Logger` type with configurable selector.